### PR TITLE
Feature/docker configs

### PR DESCRIPTION
**Pull request by danakim**
Hi @quartje ! This is the initial version of the PR. Like we discussed, you will need to add the configs to the Docker image. The paths are:

For Apache2: add it in `/etc/apache2/sites-enabled/`
For Shibboleth: overwrite the file `/etc/shibboleth/shibboleth2.xml`

### CODE REVIEW

File: .github/workflows/deploy.yml [hunk 1]
```diff
@@ -35,4 +37,34 @@ jobs:
...
+
+    - name: Set up Docker Buildx
+      uses: docker/setup-buildx-action@v2
+
+    - name: Login to GitHub Container Registry
+      uses: docker/login-action@v2
+      with:
+        registry: ghcr.io
+        username: ${{ github.repository_owner }}
+         password: ${{ secrets.GITHUB_TOKEN }}
```
Comment (quartje) [inline .github/workflows/deploy.yml:50]: A space too many here

**Review by quartje — COMMENTED** [review]
As with teams: We'd like to introduce the snapshot build process. I will create a ticket for that

File: manage-server/docker/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,3 @@
+FROM eclipse-temurin:17-jdk-alpine
```
Comment (quartje) [inline manage-server/docker/Dockerfile:1]: we're still on java 11 here. Also for the other Java apps

File: manage-gui/docker/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,2 @@
+FROM ghcr.io/openconext/openconext-basecontainers/apache2-shibboleth:latest
+ADD ./build/ /var/www/
```
Comment (quartje) [inline manage-gui/docker/Dockerfile:2]: Since we do not unpack this, COPY should do the trick

File: manage-server/docker/Dockerfile (hunk 2 above)
Comment (danakim) [reply manage-server/docker/Dockerfile:1]: @quartje - just to confirm, ALL the java apps use java 11 except the "invite" one?

File: manage-gui/docker/Dockerfile (hunk 3 above)
Comment (danakim) [reply manage-gui/docker/Dockerfile:2]: @quartje - can you please tell me a bit more, what you mean by this?

**Pull request merged**